scenario/context.py

```diff
@@ -441,48 +442,6 @@ def __init__(
     ):
         """Represents a simulated charm's execution context.
 
-        It is the main entry point to running a scenario test.
```

Comment:
Just curious why this doc block is being removed in this PR?

Reply:
It's being removed because it's duplicated in the 7.0 HEAD - I moved it to `Context.__doc__` when doing the autodoc work but must have incorrectly merged that in some branch because it ended up still in `Context.__init__.__doc__` too.

It's being removed in this PR because Pietro noticed that it was duplicated in this PR.

I can tidy it up in a separate PR rather than this drive-by if that's preferred.